flutter-for-ios.md

```diff
@@ -545,6 +545,21 @@ create a native platform integration, or use an existing
 [plugin](#plugins), such as
 [`url_launcher`](https://pub.dartlang.org/packages/url_launcher).
 
+## How do I pop back to the iOS native viewcontroller?
+
+Calling `SystemNavigator.pop()` from your Dart code invokes the
+following iOS code:
+
+```
```

Comment:
I don't know what syntax highlighter the website uses. Can we format the objective-c code?